.github/workflows/ci.yml

```diff
@@ -116,21 +116,21 @@ jobs:
       - run: npm run madge:circular
   check-docker:
     name: Docker Build
-    timeout-minutes: 15
-    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    runs-on: ubuntu-20.04
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
       - name: Set up QEMU
         id: qemu
-        uses: docker/setup-qemu-action@v1
+        uses: docker/setup-qemu-action@v2
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
+        uses: docker/setup-buildx-action@v2
       - name: Build docker image
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@v3
         with:
           context: .
-          platforms: linux/amd64
+          platforms: linux/amd64, linux/arm64/v8
```

Comment:
Since the check is only building on the `linux/amd64` platform, it's not detecting the issue. If you built across one of the arm platforms it will hang because it's an architecture that needs QEMU and buildx

I changed this to look like the release so we can tell if the versions break before needing to wait for a release.

Reply:
Makes sense